code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/community/teth/sirocco.dm

```diff
@@ -0,0 +1,141 @@
+		ABNORMALITY_WORK_REPRESSION = list(40, 20, -20, -20, -20),
+	)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/desert,
+		/datum/ego_datum/armor/desert,
+	)
+	gift_type = /datum/ego_gifts/desert
+	abnormality_origin = ABNORMALITY_ORIGIN_COMMUNITY
+
+	observation_prompt = "Why is everyone here such a bore? <br>Why doesn't anyone want to play with me? <br>\
+		A young girl's voice cries out from somewhere within the storm. <br>\
+		\"Everyone keeps staring at the floor like life's a chore...is being an adult really that bad?\" <br.\
+		\"If that's what an adult is like, I don't wanna grow up!\" <br>\
+		What do you say to it?"
+
+	observation_choices = list(
+		"You have to grow up eventually" = list(TRUE, "The storm begins to slow down. <br>\
+			\"But all of the adults here, don't they regret growing up?\" <br>\
+			Maybe. Maybe not. <br>\
+			It's better to grow and change instead of stagnating, hiding away from the real world. <br>\
+			The storm crawls to a halt. <br>\
+			\"...Maybe you're right. If no one else wants to play, maybe I should just stop playing.\" <br>\
+			You can only hope if it breaches again, it'll be less of a nuisance. <br>\
+			You leave the storm to complanate your words."),
+		"Don't worry about it" = list(FALSE, "\"Yeah! No point worrying about some lame adults!\" <br>\
+			\"Hehe! When I get out again, you gotta watch how fast I throw that lame friend of yorus!\" <br>\
+			Seems like encouraging it is only going to make things worse down the line. <br>\
+			Oh well. Maybe one day it'll tire itself out. <br>Best to leave it alone for now."),
+	)
+
+	// Prevents spawning in normal game modes
+	can_spawn = FALSE
+
+	// Work Variables
+	var/work_timer
+	var/time_to_lower = 2 MINUTES // Time to lower qliphoth
+
+	// Breach variables
+	var/cooldown_time = 3 // Cooldown between grab attempts
+	var/list/grabbed_list = list()
+	var/list_refresh_time = 30
+	var/breached_time = 5 MINUTES
+	var/lowered_breached_time = 3 MINUTES
+	var/grab_range = 3
+
+/mob/living/simple_animal/hostile/abnormality/sirocco/proc/Grabber()
+	for(var/atom/movable/A in oview(grab_range, src)) // Grab anything in our range
+		if(A && !A.anchored && !isobserver(A) && A != src)
+			A.throw_at(src, 1, 1, src, FALSE)
+	addtimer(CALLBACK(src, PROC_REF(Grabber)), cooldown_time)
+	addtimer(CALLBACK(src, PROC_REF(ThrowAround)), 2)
+
+/mob/living/simple_animal/hostile/abnormality/sirocco/proc/ThrowAround()
+	var/turf/turf_underneath = get_turf(src)
+	for(var/atom/movable/A in turf_underneath)
+		if(A && !A.anchored && !isobserver(A) && A != src && !(A in grabbed_list))
+			if(isliving(A))
+				grabbed_list += A
+			var/randomdir = rand(0, 8)
+			A.throw_at(get_edge_target_turf(src,randomdir), 5, 3, src, TRUE)
+
+/mob/living/simple_animal/hostile/abnormality/sirocco/proc/RefreshList()
+	grabbed_list = list()
+	addtimer(CALLBACK(src, PROC_REF(RefreshList)), list_refresh_time)
```

Comment:
I don't think this list is actually ever cleared when the mob dies, ya should probably make it clear itself on Destroy() to avoid hard-deletes